### PR TITLE
Feat: Ethereum transaction details

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable, but I did not test it.

File: src/protocols/aeternity/composables/aeTippingUrls.ts [hunk 1]
```diff
@@ -27,13 +27,17 @@ export function useAeTippingUrls() {
 
     const { aeActiveNetworkSettings } = useAeNetworkSettings();
 
-    const [verified, graylist] = await Promise.all([
-      fetchJson(`${aeActiveNetworkSettings.value.backendUrl}/verified`),
-      fetchJson(`${aeActiveNetworkSettings.value.backendUrl}/static/wallet/graylist`),
-    ]);
+    try {
```
Comment (martinkaintas) [inline src/protocols/aeternity/composables/aeTippingUrls.ts:30]: just did the same in my PR #2537 😅

**Review by martinkaintas — APPROVED** [review]

File: src/protocols/aeternity/composables/aeTippingUrls.ts (hunk 1 above)
Comment (peronczyk) [reply src/protocols/aeternity/composables/aeTippingUrls.ts]: Yeah, I saw this. Already merged and rebased.

File: src/popup/components/TokenAmount.vue [hunk 2]
```diff
@@ -54,18 +45,15 @@ import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
 export default defineComponent({
   props: {
     amount: { type: Number, required: true },
-    label: { type: String, default: null },
     symbol: { type: String, default: AE_SYMBOL },
-    aex9: { type: Boolean, default: false },
```
Comment (peronczyk) [inline src/popup/components/TokenAmount.vue:59]: Removing this as this does the same as `hideFiat` prop.

**Pull request merged**